recipes/flang-rt/meta.yaml

```diff
@@ -0,0 +1,111 @@
+  sha256: f8506cf5d53a5d555384a8917432db40ef8a2b89a5172d64f8a753b649f0274b
+  patches:
+    - patches/0001-avoid-baking-in-build-time-paths-to-compiler-rt-into.patch
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - flang {{ version }}.*
+  host:
+    - clangdev =={{ version }}
+    - compiler-rt =={{ version }}
+    - llvmdev =={{ version }}
+    - mlir =={{ version }}
+
+outputs:
+  - name: libflang-rt
+    build:
+      # not supported on windows yet, see
+      # https://github.com/llvm/llvm-project/issues/134186
+      skip: true  # [win]
+    files:
+      - lib/libflang_rt.runtime.so                                                          # [linux]
+      - lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so           # [linux]
+    requirements:
+      build:
+        # for strong run-exports
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+    test:
+      commands:
+        # shared library; default location
+        - test -f $PREFIX/lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.so     # [linux]
+        # symlink
+        - test -f $PREFIX/lib/libflang_rt.runtime.so                                                    # [linux]
+
+  - name: flang-rt_{{ target_platform }}
+    build:
+      noarch: generic
+    files:
+      - lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.a                # [linux]
+      - Library/lib/clang/{{ major }}/lib/x86_64-pc-windows-msvc/flang_rt.runtime.static.lib    # [win]
+    requirements:
+      host:
+        - clangdev =={{ version }}
+        - compiler-rt =={{ version }}
+        - llvmdev =={{ version }}
+        - mlir =={{ version }}
+    test:
+      commands:
+        # static library; default location
+        - test -f $PREFIX/lib/clang/{{ major }}/lib/x86_64-unknown-linux-gnu/libflang_rt.runtime.a                    # [linux]
+        - if not exist %LIBRARY_LIB%\clang\{{ major }}\lib\x86_64-pc-windows-msvc\flang_rt.runtime.static.lib exit 1  # [win]
+
+  # convenience wrapper around target-specific flang-rt
+  - name: flang-rt
+    files:                          # [linux]
+      - lib/libflang_rt.runtime.a   # [linux]
+    requirements:
+      host:
+        - {{ pin_subpackage("libflang-rt", exact=True) }}   # [linux]
+        - {{ pin_subpackage("flang-rt_" ~ target_platform, exact=True) }}
+      run:
+        - {{ pin_subpackage("libflang-rt", exact=True) }}   # [linux]
+        - {{ pin_subpackage("flang-rt_" ~ target_platform, exact=True) }}
+    test:
+      commands:
+        # symlink that's not appropriate for noarch package (but necessary for linker to find it)
+        - test -f $PREFIX/lib/libflang_rt.runtime.a     # [linux]
+
+  # compatibility output for previous name
+  - name: libflang
```

Comment:
Is this needed?

Reply:
Let's add a `run-constrained` for it and also remove the `flang-rt` convenience wrapper.

Reply:
Done.